jetstream/config.py

```diff
@@ -241,7 +241,17 @@ def last_enrollment_date_str(self) -> str:
             raise NoStartDateException(self.normandy_slug)
         return (self.start_date + dt.timedelta(days=self.proposed_enrollment)).strftime("%Y-%m-%d")
 
+    # see https://stackoverflow.com/questions/50888391/pickle-of-object-with-getattr-method-in-
+    # python-returns-typeerror-object-no
+    def __getstate__(self):
+        return vars(self)
+
+    def __setstate__(self, state):
+        vars(self).update(state)
+
     def __getattr__(self, name: str) -> Any:
+        if "experimenter_experiment" not in vars(self):
+            raise AttributeError
```

Comment:
This was the root cause for the errors I got when pickling `AnalysisConfiguration`

Reply:
Thanks for tracking that down!